tracer/src/Datadog.Trace/IAST/SensitiveData/CommandTokenizer.cs

```diff
@@ -0,0 +1,37 @@
+// <copyright file="CommandTokenizer.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Datadog.Trace.Configuration;
+
+#nullable enable
+
+namespace Datadog.Trace.Iast.SensitiveData;
+
+internal class CommandTokenizer : ITokenizer
+{
+    private static Regex _pattern = new Regex(@"^(?:\s*(?:sudo|doas|cmd|cmd.exe)\s+)?\b\S+\b(.*)", RegexOptions.Compiled | RegexOptions.IgnoreCase);
+
+    public List<Range> GetTokens(string value, IntegrationId? integrationId = null)
```

Comment:
some comments about the format you're tokenizing would be useful (in all of the tokenizers), I had to spend quite a bit of time thinking about you use `if` here and `foreach` in the others a quick explanation that regex matches all remaining tokens in this case would have made it easier to understand.

Reply:
Agree. On it.